Source/Lib/Fluxor/Store.cs

```diff
@@ -14,6 +21,10 @@ public class Store : IStore, IActionSubscriber, IDisposable
 		/// <see cref="IStore.Initialized"/>
 		public Task Initialized => InitializedCompletionSource.Task;
 
+#if NET6_0_OR_GREATER
+		private readonly IPersistenceManager? _persistenceManager;
```

Comment:
NIT: Consider following naming convention of project. Uppercase vs leading underscore for privates.

Reply:
I defer to the maintainer's preference here.